src/LevelDBStore/Plugins/Storage/LevelDB/Store.cs

```diff
@@ -7,9 +7,9 @@
 using System.Text;
 using LHelper = Neo.IO.Data.LevelDB.Helper;
 
-namespace Neo.Plugins.Storage
+namespace Neo.Plugins.Storage.LevelDB
 {
-    internal class Store : IStore
+    public class Store : IStore
```

Comment:
Why modify this file?

Reply:
I want to use leveldb in `StateStorage`.

Reply:
I think we can use `IStore`. What do you think?

Reply:
`StateService` is already a plugin, how to determine `IStore`? Besides, `RpcNep17Tracker` and `ApplicationLog` both use `levelDB` to store data.

Reply:
Or I use `Neo.IO.Data.LevelDB`.

Reply:
Do you want to use the same store as `Blockchain`, or use a separate store?

Reply:
Use a seperate one.

Reply:
Please see neo-project/neo#2172

Reply:
I will update this pr after https://github.com/neo-project/neo/pull/2172.

Reply:
We can move on.

Reply:
Done.